charts/nexus-repository-manager/values.yaml

```diff
@@ -45,8 +45,13 @@ nexus:
   # The ports should only be changed if the nexus image uses a different port
   nexusPort: 8081
 
+  # Default the pods UID and GID to match the nexus3 container.
+  # Customize or remove these values from the securityContext as appropriate for
+  # your deployment environment.
   securityContext:
-    fsGroup: 2000
+    runAsUser: 200
+    runAsGroup: 200
+    fsGroup: 200
```

Comment:
these values as a default will match the container now. It may help in security environments that aren't even allowed to think about running a root container, so this explicitly starts it up with the right IDs.

Reply:
Can we also update the heap size / memory settings to [minimum recommended](https://help.sonatype.com/repomanager3/product-information/system-requirements#SystemRequirements-MemoryRequirements) values as per our doc?

-Xms2703M
-Xmx2703M
-XX:MaxDirectMemorySize=2703M

Reply:
yeah, i'll create a jira for it.